New package: RFFT v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: RFFT
- Repository: https://github.com/JuliaMath/RFFT.jl
- Created by: @IanButterworth
- Version: v1.0.0
- Commit: 5402b9677a35996502faf6e8587042623e235fc0
- Reviewed by: @IanButterworth
- Reference: https://github.com/JuliaMath/RFFT.jl/commit/5402b9677a35996502faf6e8587042623e235fc0#commitcomment-141481440
- Description: In-place real FFTs for Julia
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6af9cd2a2a39d7785f30e6b1fd92edb2e0b97e0cf2c9f0b7b74cbebb0f06e406ce607ff7cf56413fcce0c7f100867ec2f5e511290b88f9a957e1f197dcf47368d119fc45f6b4d67385ac4918d405fe57fb376b297fd68ea868e23e332cb4ece904e56e9bf87ded264890da19b731e71f9e4d556c711ab89150cc1584339acaf7e0cb5fd46b9d2da4407dc1f42466f38842905c7ca5ef42b1a3204820d6989efcd3298b7019f28318aa7a196065f18d54b -->

IanButterworth commented:
FYI @timholy this is part of the plan discussed [here](https://github.com/HolyLab/RFFT.jl/pull/17#issuecomment-1999889289) and [here](https://github.com/HolyLab/RFFT.jl/pull/17#issuecomment-2047178576)

JuliaRegistrator committed:
New package: RFFT v1.0.0

UUID: 3645faec-0534-4e91-afef-021db0981eec
Repo: https://github.com/JuliaMath/RFFT.jl.git
Tree: 45f2756640780c259d2aaeb3cb742a53aaf38396

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

timholy commented:
Superseded by #105861
